documentation/api/commands.markdown

```diff
@@ -90,9 +90,23 @@ payload of this command.
 
 ### "deactivate node", version 2
 
-The payload is expected to be the certname of a node, as a raw JSON
-string, which will be deactivated effective as of the time the command
-is *processed*.
+The payload is a JSON map, formatted as follows:
+
+     {
+      "certname": <string>,
+      "producer_timestamp": <datetime>
+     }
```

Comment:
I'm not sure. I'm mostly ok leaving this as is, as it's still pretty small, but could probably be convinced otherwise.